src/stores/tenant.rs

```diff
@@ -218,9 +218,12 @@ impl Tenant {
                         &self.apns_team_id,
                     ) {
                         (Some(topic), Some(pkcs8_pem), Some(key_id), Some(team_id)) => {
-                            let mut reader = BufReader::new(pkcs8_pem.as_bytes());
+                            info!("apns token (p8) provider is matched");
+                            let p8_token = base64::engine::general_purpose::STANDARD
+                                .decode(pkcs8_pem)
+                                .expect("p8 token is not a valid base64 string");
```

Comment:
Use `?` instead of `.expect()`

Reply:
When using `?` here we will miss the context, if we use `expect` we can pass additional context right to the error, so the error will be like `p8 token is not a valid base64 string: Bad character at $%!`.

Reply:
Removing it because this can't be handled properly in this behavior ((